stor/tests/test_dx.py

```diff
@@ -901,13 +901,18 @@ def test_on_file(self):
         dx_p = DXPath('dx://' + self.project + ':/temp_file.txt')
         actual_url = dx_p.temp_url()
         self.assertIn('dl.dnanex.us', actual_url)
+        self.assertIn('temp_file.txt', actual_url)
```

Comment:
do you plan to add a test with `file_proxy_url` set?

Reply:
added test